jest.config.js

```diff
@@ -1,3 +1,28 @@
+const packagesToTranspile = [
+  'lodash-es',
+  '@aeternity/aepp-sdk',
+  '@aeternity/hd-wallet',
+];
+
 module.exports = {
-  preset: '@vue/cli-plugin-unit-jest',
```

Comment:
Maybe we can stick with some preset which is more suitable for us instead of manually writing all rules